webhook_features/src/features/summary_comment.rs

```diff
@@ -296,7 +296,10 @@ lazy_static! {
         (r"\b(Approach NACK)\b", AckType::ApproachNack),
         (r"\b(NACK)\b", AckType::ConceptNack),
         (r"\b(Concept ACK)\b", AckType::ConceptAck),
-        (r"(ACK)(?:\s*)([0-9a-f]{6,40})\b", AckType::Ack),
+        (
+            r"(ACK)(?:\s*)(((https://github\.com/.*/commits?/)|\[)?([0-9a-f]{6,40}))\b",
```

Comment:
```suggestion
            r"(ACK)(?:.*?)([0-9a-f]{6,40})\b",
```

Reply:
nit: What about just taking the first commit hash that follows?

Reply:
I'm a bit afraid of comments like "I would love to ACK but I don't understand the code in `1235678`".

Reply:
Well, it is never possible to be free of false-positives. There is also "Almost ACK 1235678, if the bug is fixed"

Reply:
Sure, pushed.